scripts/migrate.sh

```
@@ -0,0 +1,61 @@
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+
+cd "$PROJECT_ROOT"
+
+echo "🔍 Checking migration status..."
+
+# Function to apply migration via MCP
+apply_via_mcp() {
+    local migration_file=$1
+    local migration_name=$(basename "$migration_file" .sql)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)